src/elm/Page/Register.elm

```diff
@@ -27,12 +29,10 @@ import Result
 import Route
 import Session.Guest as Guest exposing (External(..))
 import Session.Shared exposing (Shared, Translators)
+import Set exposing (Set)
 import UpdateResult as UR
-import Validate
+import View.Components
 import View.Feedback as Feedback
-import View.Form
-import View.Form.Checkbox as Checkbox
-import View.Form.Input as Input
 
 
```

Comment:
Same thing as the other page, something's up with the validation trigger, should happen only if something has changed, at least on the first time

https://user-images.githubusercontent.com/1082127/150032668-22dd2736-5976-4b4e-a7ce-19e8171711a6.mov